source/se/failure.html

```diff
@@ -18,13 +18,13 @@
 			</div>
 		</header>
 		<div class="Container">
-			<h1>Något gick fel!</h1>
-			<p>
-				Gå tillbaka till vår beställningssidan och testa igen.
-			</p>
-			<a href="/estore-express/se/">
-				<button class="Button Button--primary">Tillbaka</button>
-			</a>
+			<div class="Result">
+				<h1>Något gick fel!</h1>
+				<p>Gå tillbaka till vår beställningssidan och testa igen.</p>
```

Comment:
> Gå tillbaka till vår beställningssidan och testa igen.

Shouldn't it be `Gå tillbaka till vår beställningssida` or `Gå tillbaka till bestälningssidan`?